### ISSUE TITLE
2025-01-16, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#53]
# Today's Arxiv Papers

(1) [Multimodal LLMs Can Reason about Aesthetics in Zero-Shot](http://arxiv.org/abs/2501.09012v1), 2025-01-15

(2) [Aegis2.0: A Diverse AI Safety Dataset and Risks Taxonomy for Alignment of LLM Guardrails](http://arxiv.org/abs/2501.09004v1), 2025-01-15

(3) [Development and Validation of the Provider Documentation Summarization Quality Instrument for Large Language Models](http://arxiv.org/abs/2501.08977v1), 2025-01-15

(4) [Learning to Extract Cross-Domain Aspects and Understanding Sentiments Using Large Language Models](http://arxiv.org/abs/2501.08974v1), 2025-01-15

(5) [Analyzing the Ethical Logic of Six Large Language Models](http://arxiv.org/abs/2501.08951v1), 2025-01-15

(6) [Applying General Turn-taking Models to Conversational Human-Robot Interaction](http://arxiv.org/abs/2501.08946v1), 2025-01-15

(7) [Physical AI Agents: Integrating Cognitive Intelligence with Real-World Action](http://arxiv.org/abs/2501.08944v1), 2025-01-15

(8) [Disentangling Exploration of Large Language Models by Optimal Exploitation](http://arxiv.org/abs/2501.08925v1), 2025-01-15

(9) [GenAI Content Detection Task 3: Cross-Domain Machine-Generated Text Detection Challenge](http://arxiv.org/abs/2501.08913v1), 2025-01-15

(10) [Leveraging Large Language Models as Knowledge-Driven Agents for Reliable Retrosynthesis Planning](http://arxiv.org/abs/2501.08897v1), 2025-01-15

(11) [ToMATO: Verbalizing the Mental States of Role-Playing LLMs for Benchmarking Theory of Mind](http://arxiv.org/abs/2501.08838v1), 2025-01-15

(12) [How Developers Interact with AI: A Taxonomy of Human-AI Collaboration in Software Engineering](http://arxiv.org/abs/2501.08774v1), 2025-01-15

(13) [Enhanced Large Language Models for Effective Screening of Depression and Anxiety](http://arxiv.org/abs/2501.08769v1), 2025-01-15

(14) [Leveraging LLM Agents for Translating Network Configurations](http://arxiv.org/abs/2501.08760v1), 2025-01-15

(15) [The Inherent Limits of Pretrained LLMs: The Unexpected Convergence of Instruction Tuning and In-Context Learning Capabilities](http://arxiv.org/abs/2501.08716v1), 2025-01-15

(16) [Knowledge Graph-based Retrieval-Augmented Generation for Schema Matching](http://arxiv.org/abs/2501.08686v1), 2025-01-15

(17) [Augmenting Smart Contract Decompiler Output through Fine-grained Dependency Analysis and LLM-facilitated Semantic Recovery](http://arxiv.org/abs/2501.08670v1), 2025-01-15

(18) [MAGNET: Augmenting Generative Decoders with Representation Learning and Infilling Capabilities](http://arxiv.org/abs/2501.08648v1), 2025-01-15

(19) [Reassessing the Role of Chain-of-Thought in Sentiment Analysis: Insights and Limitations](http://arxiv.org/abs/2501.08641v1), 2025-01-15

(20) [SWSC: Shared Weight for Similar Channel in LLM](http://arxiv.org/abs/2501.08631v1), 2025-01-15

(21) [Disjoint Processing Mechanisms of Hierarchical and Linear Grammars in Large Language Models](http://arxiv.org/abs/2501.08618v1), 2025-01-15

(22) [Assessing the Alignment of FOL Closeness Metrics with Human Judgement](http://arxiv.org/abs/2501.08613v1), 2025-01-15

(23) [Monte Carlo Tree Search for Comprehensive Exploration in LLM-Based Automatic Heuristic Design](http://arxiv.org/abs/2501.08603v1), 2025-01-15

(24) [AutoRestTest: A Tool for Automated REST API Testing Using LLMs and MARL](http://arxiv.org/abs/2501.08600v1), 2025-01-15

(25) [LlamaRestTest: Effective REST API Testing with Small Language Models](http://arxiv.org/abs/2501.08598v1), 2025-01-15

(26) [LoRS: Efficient Low-Rank Adaptation for Sparse Large Language Model](http://arxiv.org/abs/2501.08582v1), 2025-01-15

(27) [What Limits LLM-based Human Simulation: LLMs or Our Design?](http://arxiv.org/abs/2501.08579v1), 2025-01-15

(28) [Information Entropy Invariance: Enhancing Length Extrapolation in Attention Mechanisms](http://arxiv.org/abs/2501.08570v1), 2025-01-15

(29) [Adaptive Sampled Softmax with Inverted Multi-Index: Methods, Theory and Applications](http://arxiv.org/abs/2501.08563v1), 2025-01-15

(30) [LAMS: LLM-Driven Automatic Mode Switching for Assistive Teleoperation](http://arxiv.org/abs/2501.08558v1), 2025-01-15

(31) [The Devil is in Temporal Token: High Quality Video Reasoning Segmentation](http://arxiv.org/abs/2501.08549v1), 2025-01-15

(32) [Knowledge prompt chaining for semantic modeling](http://arxiv.org/abs/2501.08540v1), 2025-01-15

(33) [Doc-Guided Sent2Sent++: A Sent2Sent++ Agent with Doc-Guided memory for Document-level Machine Translation](http://arxiv.org/abs/2501.08523v1), 2025-01-15

(34) [Quantifying the Importance of Data Alignment in Downstream Model Performance](http://arxiv.org/abs/2501.08496v1), 2025-01-14

(35) [The Theater Stage as Laboratory: Review of Real-Time Comedy LLM Systems for Live Performance](http://arxiv.org/abs/2501.08474v1), 2025-01-14

(36) [Towards Zero-Shot & Explainable Video Description by Reasoning over Graphs of Events in Space and Time](http://arxiv.org/abs/2501.08460v1), 2025-01-14

(37) [Large Language Models For Text Classification: Case Study And Comprehensive Review](http://arxiv.org/abs/2501.08457v1), 2025-01-14

(38) [Tag&Tab: Pretraining Data Detection in Large Language Models Using Keyword-Based Membership Inference Attack](http://arxiv.org/abs/2501.08454v1), 2025-01-14

(39) [SEAL: Speaker Error Correction using Acoustic-conditioned Large Language Models](http://arxiv.org/abs/2501.08421v1), 2025-01-14

(40) [Ensemble of Large Language Models for Curated Labeling and Rating of Free-text Data](http://arxiv.org/abs/2501.08413v1), 2025-01-14

(41) [OptiChat: Bridging Optimization Models and Practitioners with Large Language Models](http://arxiv.org/abs/2501.08406v1), 2025-01-14

(42) [PokerBench: Training Large Language Models to become Professional Poker Players](http://arxiv.org/abs/2501.08328v1), 2025-01-14

(43) [Omni-RGPT: Unifying Image and Video Region-level Understanding via Token Marks](http://arxiv.org/abs/2501.08326v1), 2025-01-14

(44) [ADAM-1: AI and Bioinformatics for Alzheimer's Detection and Microbiome-Clinical Data Integrations](http://arxiv.org/abs/2501.08324v1), 2025-01-14

(45) [Exploring Robustness of Multilingual LLMs on Real-World Noisy Data](http://arxiv.org/abs/2501.08322v1), 2025-01-14

(46) [Enhancing Automated Interpretability with Output-Centric Feature Descriptions](http://arxiv.org/abs/2501.08319v1), 2025-01-14

(47) [HALoGEN: Fantastic LLM Hallucinations and Where to Find Them](http://arxiv.org/abs/2501.08292v1), 2025-01-14

(48) [AfriHate: A Multilingual Collection of Hate Speech and Abusive Language Datasets for African Languages](http://arxiv.org/abs/2501.08284v2), 2025-01-14

(49) [LLaVA-ST: A Multimodal Large Language Model for Fine-Grained Spatial-Temporal Understanding](http://arxiv.org/abs/2501.08282v1), 2025-01-14

(50) [Exploring Robustness of LLMs to Sociodemographically-Conditioned Paraphrasing](http://arxiv.org/abs/2501.08276v1), 2025-01-14

(51) [Addressing the sustainable AI trilemma: a case study on LLM agents and RAG](http://arxiv.org/abs/2501.08262v1), 2025-01-14

(52) [Towards Best Practices for Open Datasets for LLM Training](http://arxiv.org/abs/2501.08365v1), 2025-01-14

(53) [Eliciting In-context Retrieval and Reasoning for Long-context Large Language Models](http://arxiv.org/abs/2501.08248v1), 2025-01-14

(54) [Text-Diffusion Red-Teaming of Large Language Models: Unveiling Harmful Behaviors with Proximity Constraints](http://arxiv.org/abs/2501.08246v1), 2025-01-14

(55) [Engineering LLM Powered Multi-agent Framework for Autonomous CloudOps](http://arxiv.org/abs/2501.08243v1), 2025-01-14

(56) [Investigating Energy Efficiency and Performance Trade-offs in LLM Inference Across Tasks and DVFS Settings](http://arxiv.org/abs/2501.08219v1), 2025-01-14

(57) [ASTRID -- An Automated and Scalable TRIaD for the Evaluation of RAG-based Clinical Question Answering Systems](http://arxiv.org/abs/2501.08208v1), 2025-01-14

(58) [ArithmAttack: Evaluating Robustness of LLMs to Noisy Context in Math Problem Solving](http://arxiv.org/abs/2501.08203v1), 2025-01-14

(59) [CWEval: Outcome-driven Evaluation on Functionality and Security of LLM Code Generation](http://arxiv.org/abs/2501.08200v1), 2025-01-14

(60) [OpenCSG Chinese Corpus: A Series of High-quality Chinese Datasets for LLM Training](http://arxiv.org/abs/2501.08197v1), 2025-01-14

(61) [PRESERVE: Prefetching Model Weights and KV-Cache in Distributed LLM Serving](http://arxiv.org/abs/2501.08192v1), 2025-01-14

(62) [A Multi-Modal AI Copilot for Single-Cell Analysis with Instruction Following](http://arxiv.org/abs/2501.08187v2), 2025-01-14

(63) [Potential and Perils of Large Language Models as Judges of Unstructured Textual Data](http://arxiv.org/abs/2501.08167v1), 2025-01-14

(64) [I Can Find You in Seconds! Leveraging Large Language Models for Code Authorship Attribution](http://arxiv.org/abs/2501.08165v1), 2025-01-14

(65) [Refusal Behavior in Large Language Models: A Nonlinear Perspective](http://arxiv.org/abs/2501.08145v1), 2025-01-14

(66) [Consistency of Responses and Continuations Generated by Large Language Models on Social Media](http://arxiv.org/abs/2501.08102v2), 2025-01-14

(67) [Hierarchical Autoscaling for Large Language Model Serving with Chiron](http://arxiv.org/abs/2501.08090v1), 2025-01-14

(68) [CuAsmRL: Optimizing GPU SASS Schedules via Deep Reinforcement Learning](http://arxiv.org/abs/2501.08071v1), 2025-01-14

(69) [A Roadmap to Guide the Integration of LLMs in Hierarchical Planning](http://arxiv.org/abs/2501.08068v1), 2025-01-14

(70) [Exploring Narrative Clustering in Large Language Models: A Layerwise Analysis of BERT](http://arxiv.org/abs/2501.08053v1), 2025-01-14

(71) [TriAdaptLoRA: Brain-Inspired Triangular Adaptive Low-Rank Adaptation for Parameter-Efficient Fine-Tuning](http://arxiv.org/abs/2501.08008v1), 2025-01-14

(72) [LLM-Ehnanced Holonic Architecture for Ad-Hoc Scalable SoS](http://arxiv.org/abs/2501.07992v1), 2025-01-14

(73) [Facial Dynamics in Video: Instruction Tuning for Improved Facial Expression Perception and Contextual Awareness](http://arxiv.org/abs/2501.07978v1), 2025-01-14

(74) [Zero-shot Video Moment Retrieval via Off-the-shelf Multimodal Large Language Models](http://arxiv.org/abs/2501.07972v1), 2025-01-14

(75) [Self-Instruct Few-Shot Jailbreaking: Decompose the Attack into Pattern and Behavior Learning](http://arxiv.org/abs/2501.07959v1), 2025-01-14

(76) [Advice for Diabetes Self-Management by ChatGPT Models: Challenges and Recommendations](http://arxiv.org/abs/2501.07931v1), 2025-01-14

(77) [Gandalf the Red: Adaptive Security for LLMs](http://arxiv.org/abs/2501.07927v1), 2025-01-14

(78) [Large Language Model Interface for Home Energy Management Systems](http://arxiv.org/abs/2501.07919v1), 2025-01-14

(79) [Leveraging Metamemory Mechanisms for Enhanced Data-Free Code Generation in LLMs](http://arxiv.org/abs/2501.07892v1), 2025-01-14

(80) [ReARTeR: Retrieval-Augmented Reasoning with Trustworthy Process Rewarding](http://arxiv.org/abs/2501.07861v1), 2025-01-14

(81) [Hierarchical Repository-Level Code Summarization for Business Applications Using Local LLMs](http://arxiv.org/abs/2501.07857v1), 2025-01-14

(82) [Optimizing Language Models for Grammatical Acceptability: A Comparative Study of Fine-Tuning Techniques](http://arxiv.org/abs/2501.07853v1), 2025-01-14

(83) [Unveiling Provider Bias in Large Language Models for Code Generation](http://arxiv.org/abs/2501.07849v1), 2025-01-14

(84) [Reasoning with Graphs: Structuring Implicit Knowledge to Enhance LLMs Reasoning](http://arxiv.org/abs/2501.07845v1), 2025-01-14

(85) [A Driver Advisory System Based on Large Language Model for High-speed Train](http://arxiv.org/abs/2501.07837v1), 2025-01-14

(86) [Flow: A Modular Approach to Automated Agentic Workflow Generation](http://arxiv.org/abs/2501.07834v1), 2025-01-14

(87) [Real-time Verification and Refinement of Language Model Text Generation](http://arxiv.org/abs/2501.07824v1), 2025-01-14

(88) [3UR-LLM: An End-to-End Multimodal Large Language Model for 3D Scene Understanding](http://arxiv.org/abs/2501.07819v1), 2025-01-14

(89) [A Multi-Encoder Frozen-Decoder Approach for Fine-Tuning Large Language Models](http://arxiv.org/abs/2501.07818v1), 2025-01-14

(90) [Agent-Centric Projection of Prompting Techniques and Implications for Synthetic Training Data for Large Language Models](http://arxiv.org/abs/2501.07815v1), 2025-01-14

(91) [Talk to Right Specialists: Routing and Planning in Multi-agent System for Question Answering](http://arxiv.org/abs/2501.07813v1), 2025-01-14

(92) [CodeCoR: An LLM-Based Self-Reflective Multi-Agent Framework for Code Generation](http://arxiv.org/abs/2501.07811v1), 2025-01-14

(93) [Visual Language Models as Operator Agents in the Space Domain](http://dx.doi.org/10.2514/6.2025-1543), 2025-01-14

(94) [Parameter-Inverted Image Pyramid Networks for Visual Perception and Multimodal Understanding](http://arxiv.org/abs/2501.07783v1), 2025-01-14

(95) [Large Language Models for Knowledge Graph Embedding Techniques, Methods, and Challenges: A Survey](http://arxiv.org/abs/2501.07766v1), 2025-01-14
